website/docs/r/disk_encryption_set.html.markdown

```diff
@@ -0,0 +1,140 @@
+---
+subcategory: "Compute"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_disk_encryption_set"
+sidebar_current: "docs-azurerm-resource-disk-encryption-set"
+description: |-
+  Manage Azure DiskEncryptionSet instance.
+---
+
+# azurerm_disk_encryption_set
+
+Manage an Azure DiskEncryptionSet instance.
+
+-> **NOTE:** The DiskEncryptionSet service is currently in public preview and are only available in a limited set of regions. 

```

Comment:
can we make the phrasing consistent with other resources:

```suggestion
-> **NOTE:** Disk Encryption Sets are currently in Public Preview and are only available in a limited set of regions. 
```

how do users opt-into this?

Reply:
Let me ask the service team and then fill this information in.